js/window.js

```diff
@@ -206,4 +206,31 @@ form.addEventListener('submit', (event) => {
   if (!emailValid) {
     event.preventDefault();
   }
-});
\ No newline at end of file
+});
+/* save form data */
+const userName = form.elements.name;
+const userEmail = form.elements.email;
+const userMessage = form.elements.message;
+function populateStorage() {
+  const userInput = [form.elements.name.value, form.elements.email.value,
```

Comment:
- [ ] As requested by the previous code reviewer, which also a requirement for this project, please make sure you create a single JavaScript object with all the data from the entire form and save it in local storage. You have an array of inputs, please kindly change that. Please this [article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Working_with_Objects) will be of help. 
- [ ] Your data is not been saved to the local storage, when you start filling the form and refresh(reload) the page, the data is lost, meanwhile it is supposed to be saved to the local storage, please add a change event listener to the form to track the changes in the input fields and save the data to local storage. Checkout this [article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Working_with_Objects).


![tahir22](https://user-images.githubusercontent.com/65852621/136540556-62600c3d-6f3a-4eb5-b2bc-f184e1966c86.png)